arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1936 @@
+      "description": "Contains the user facing view of the properties of the provisioned event hub used by the Iot Hub",
+      "type": "object",
+      "properties": {
+        "retentionTimeInDays": {
+          "format": "int64",
+          "description": "Gets or sets the retention time in days.",
+          "type": "integer"
+        },
+        "partitionCount": {
+          "format": "int32",
+          "description": "Gets or sets the partition count.",
+          "type": "integer"
+        },
+        "partitionIds": {
+          "description": "Gets or sets the partition ids.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "path": {
+          "description": "Gets or sets the path.",
+          "type": "string"
+        },
+        "endpoint": {
+          "description": "Gets or sets the endpoint.",
+          "type": "string"
+        },
+        "internalAuthorizationPolicies": {
+          "description": "Gets or sets the internal authorization rules.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SharedAccessAuthorizationRule"
+          }
+        },
+        "authorizationPolicies": {
+          "description": "Gets or sets the authorization rules.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SharedAccessAuthorizationRule"
+          }
+        }
+      }
+    },
+    "StorageEndpointProperties": {
+      "description": "Contains the user facing view of the properties of the Storage Endpoint provided by the customer for file upload",
+      "type": "object",
+      "properties": {
+        "sasTtlAsIso8601": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "connectionString": {
+          "description": "Account key credentials for storage account selected by customer for uploading files",
+          "type": "string"
+        },
+        "containerName": {
+          "description": "Root container name where all files will be uploaded",
+          "type": "string"
+        }
+      }
+    },
+    "MessagingEndpointProperties": {
+      "description": "Contains the user facing view of the properties of the Messaging Endpoints used by this IoT Hub",
+      "type": "object",
+      "properties": {
+        "lockDurationAsIso8601": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "ttlAsIso8601": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "maxDeliveryCount": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "CloudToDeviceProperties": {
+      "description": "Contains user facing view of Iot Hub Cloud-To-Device messaging properties",
+      "type": "object",
+      "properties": {
+        "maxDeliveryCount": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "defaultTtlAsIso8601": {
```

Comment:
Missing description. (Descriptions are important, they are used to generate the help documentation for the APIs)